lib/commands/new/project-template.js

```diff
@@ -28,6 +28,8 @@ exports.ProjectTemplate = class {
       paths: {}
     });
 
+    this.model.serve = this.createServeConfig();
```

Comment:
Perhaps a better place to set this would be in [create](https://github.com/casion/cli/blob/6fbfb3daf6d688da69dce300cbf9fc97b857a1da/lib/commands/new/project-template.js#L272) where we set `baseDir` as well